mmcls/models/backbones/efficientnet_v2.py

```diff
@@ -0,0 +1,229 @@
+        'm': [[3, 3, 1, 1, 24, 24, 0, -1], [5, 3, 2, 4, 24, 48, 0, 0],
+              [5, 3, 2, 4, 48, 80, 0, 0], [7, 3, 2, 4, 80, 160, 0.25, 1],
+              [14, 3, 1, 6, 160, 176, 0.25, 1],
+              [18, 3, 2, 6, 176, 304, 0.25, 1],
+              [5, 3, 1, 6, 304, 512, 0.25, 1]],
+        'l': [[4, 3, 1, 1, 32, 32, 0, -1], [7, 3, 2, 4, 32, 64, 0, 0],
+              [7, 3, 2, 4, 64, 96, 0, 0], [10, 3, 2, 4, 96, 192, 0.25, 1],
+              [19, 3, 1, 6, 192, 224, 0.25, 1],
+              [25, 3, 2, 6, 224, 384, 0.25, 1],
+              [7, 3, 1, 6, 384, 640, 0.25, 1]],
+        'xl': [[4, 3, 1, 1, 32, 32, 0, -1], [8, 3, 2, 4, 32, 64, 0, 0],
+               [8, 3, 2, 4, 64, 96, 0, 0], [16, 3, 2, 4, 96, 192, 0.25, 1],
+               [24, 3, 1, 6, 192, 256, 0.25, 1],
+               [32, 3, 2, 6, 256, 512, 0.25, 1],
+               [8, 3, 1, 6, 512, 640, 0.25, 1]],
+        'b0': [[1, 3, 1, 1, 32, 16, 0, -1], [2, 3, 2, 4, 16, 32, 0, 0],
+               [2, 3, 2, 4, 32, 48, 0, 0], [3, 3, 2, 4, 48, 96, 0.25, 1],
+               [5, 3, 1, 6, 96, 112, 0.25, 1],
+               [8, 3, 2, 6, 112, 192, 0.25, 1]],
+        'b1': [[2, 3, 1, 1, 32, 16, 0, -1], [3, 3, 2, 4, 16, 32, 0, 0],
+               [3, 3, 2, 4, 32, 48, 0, 0], [4, 3, 2, 4, 48, 96, 0.25, 1],
+               [6, 3, 1, 6, 96, 112, 0.25, 1],
+               [9, 3, 2, 6, 112, 192, 0.25, 1]],
+        'b2': [[2, 3, 1, 1, 32, 16, 0, -1], [3, 3, 2, 4, 16, 32, 0, 0],
+               [3, 3, 2, 4, 32, 56, 0, 0], [4, 3, 2, 4, 56, 104, 0.25, 1],
+               [6, 3, 1, 6, 104, 120, 0.25, 1],
+               [10, 3, 2, 6, 120, 208, 0.25, 1]],
+        'b3': [[2, 3, 1, 1, 40, 16, 0, -1], [3, 3, 2, 4, 16, 40, 0, 0],
+               [3, 3, 2, 4, 40, 56, 0, 0], [5, 3, 2, 4, 56, 112, 0.25, 1],
+               [7, 3, 1, 6, 112, 136, 0.25, 1],
+               [12, 3, 2, 6, 136, 232, 0.25, 1]]
+    }
+
+    def __init__(self,
+                 arch: str = 's',
+                 drop_path_rate: float = 0.,
+                 out_indices: Tuple = (-1,),
+                 frozen_stages: int = 0,
+                 conv_cfg=dict(type='Conv2dAdaptivePadding'),
+                 norm_cfg=dict(type='BN', eps=1e-3, momentum=0.1),
+                 act_cfg=dict(type='Swish'),
+                 norm_eval: bool = False,
+                 with_cp: bool = False,
+                 init_cfg=[
+                     dict(type='Kaiming', layer='Conv2d'),
+                     dict(
+                         type='Constant',
+                         layer=['_BatchNorm', 'GroupNorm'],
+                         val=1)
+                 ]):
+        super(EfficientNetV2, self).__init__(init_cfg)
+        assert arch in self.arch_settings, \
+            f'"{arch}" is not one of the arch_settings ' \
+            f'({", ".join(self.arch_settings.keys())})'
+        self.arch = self.arch_settings[arch]
+        self.drop_path_rate = drop_path_rate
+        self.out_indices = out_indices
```

Comment:
it would be more robust if add these lines of codes (maybe should modify a little):
https://github.com/open-mmlab/mmclassification/blob/458ac4c89ae18ae49750b6e66a30f1a08720b77e/mmcls/models/backbones/mobileone.py#L419-L430

Reply:
received